drivers/sensor/pni/rm3100/rm3100.h

```diff
@@ -50,9 +50,7 @@ struct rm3100_stream {
 		struct {
 			bool drdy : 1;
 		} enabled;
-		struct {
-			enum sensor_stream_data_opt drdy;
-		} opt;
+		enum sensor_stream_data_opt opt;
```

Comment:
> Rename sensor_stream_data_opt drdy to just opt to better represent the variable containing other states besides drdy.


Curious on what are the other states that we'd be able to better represent by renaming this to opt.